gokart/mypy.py

```diff
@@ -55,7 +61,50 @@
 PARAMETER_TMP_MATCHER: Final = re.compile(r'^\w*Parameter$')
 
 
+@dataclass
+class TaskOnKartPluginOptions:
+    # Whether to error on missing parameters in the constructor.
+    # Some projects use luigi.Config to set parameters, which does not require parameters to be explicitly passed to the constructor.
+    error_on_missing_parameters: bool = False
+
+    @classmethod
+    def _parse_toml(cls, config_file: str) -> Dict[str, Any]:
+        if sys.version_info >= (3, 11):
+            import tomllib as toml_
+        else:
+            try:
+                import tomli as toml_
+            except ImportError:  # pragma: no cover
+                warnings.warn('install tomli to parse pyproject.toml under Python 3.10', stacklevel=1)
+                return {}
+
+        with open(config_file, 'rb') as f:
+            return toml_.load(f)
+
+    @classmethod
+    def parse_config_file(cls, config_file: Optional[str]) -> 'TaskOnKartPluginOptions':
+        if not config_file:
+            return cls()
+
+        # TODO: support other configuration file formats if necessary.
+        if not config_file.endswith('.toml'):
+            warnings.warn('gokart mypy plugin can be configured by pyproject.toml', stacklevel=1)
+            return cls()
+
+        config = cls._parse_toml(config_file)
+        gokart_plugin_config = config.get('tool', {}).get('gokart-mypy', {})
+
+        error_on_missing_parameters = gokart_plugin_config.get('error_on_missing_parameters', False)
```

Comment:
I think parameter name should be FINAL variable.

Reply:
I introduce Enum for parameters